src/main/scala/com/microsoft/azure/cosmosdb/spark/CosmosDBConnection.scala

```diff
@@ -116,11 +118,26 @@ private[spark] case class CosmosDBConnection(config: Config) extends LoggingTrai
     bulkImporter
   }
 
+  def setDefaultClientRetryPolicy: Unit = {
+    if (documentClient != null) {
+      documentClient.getConnectionPolicy().getRetryOptions().setMaxRetryAttemptsOnThrottledRequests(9);
```

Comment:
we should get the default values from a new connection policy instead of a hard coded values here